console/console-init/ui/src/components/ToggleIcon/ToggleIcon.tsx

```diff
@@ -4,10 +4,16 @@
  */
 
 import React from "react";
-import { ToggleOffIcon, ToggleOnIcon, IconSize } from "@patternfly/react-icons";
+import {
+  IconSize,
+  ToggleOnIcon as OnIcon,
+  ToggleOffIcon as OffIcon
+} from "@patternfly/react-icons";
 
 export type OneOf<T, K extends keyof T> = T[K];
 
+export type IconType = React.SFC<any>;
```

Comment:
It is showing a deprecated warning message and a suggestion to use the `FunctionComponent` type. Is using React.FC feasible here?

Reply:
ok. I'll change